src/standalone/standard.rs

```diff
@@ -137,34 +123,68 @@ impl StandardDocumentOps for StandaloneStandardClient {
 
         Ok(encrypted_doc)
     }
+
     async fn decrypt(
         &self,
         encrypted_document: EncryptedDocument,
         metadata: &AlloyMetadata,
     ) -> Result<PlaintextDocument, AlloyError> {
-        let dek =
-            Self::decrypt_document_dek(encrypted_document.edek, &self.config.secrets, metadata)?;
+        let (_, dek) = Self::decrypt_document_dek(
```

Comment:
I know it's sort of becoming common across this codebase at this point, but there's not really much of a cost to using structs with named fields. It was intuitive to me reading this as to why `decrypt_document_dek` was now returning the `dek` and something else. It's still unclear to me if it makes most sense to rename the function or name the return type. Probably the function, since the type in a lot of these cases would be something like `KeyIdAndDek`, which isn't generally an improvement over the tuple.

In general in other places I find `something.content` a lot more readable than `something.0` for everything but newtypes.

Reply:
A chain of events ended with not needing it to return the KeyId, so I undid this change. In general I agree though